### PR TITLE
Added notifications when flagging failed

### PR DESCRIPTION
**Pull request by PepeFranco**
## What does this PR do?

It adds notifications to user when they tried to flag a comment/user and the action fails

## How do I test this PR?

Have the api that responds to flags not started and try to flag a comment/user

### CODE REVIEW

File: client/talk-plugin-flags/components/FlagButton.js [hunk 1]
```diff
@@ -99,6 +99,7 @@ export default class FlagButton extends Component {
             }
           })
           .catch((err) => {
+            this.props.notify('error', t('error.COMMENT_FLAG_FAILED'));
```
Comment (alinhle) [inline client/talk-plugin-flags/components/FlagButton.js]: Can you change these lines to `this.props.notify('error', getErrorMessages(err));` and add an `import {getErrorMessages} from 'coral-framework/utils';` at the top?

**Review by alinhle — CHANGES_REQUESTED** [review]

File: client/talk-plugin-flags/components/FlagButton.js (hunk 1 above)
Comment (PepeFranco) [reply client/talk-plugin-flags/components/FlagButton.js]: Sure, will do

**Comment by PepeFranco** [comment]
Just did the change as requested by @cvle

**Review by alinhle — APPROVED** [review]
@PepeFranco Perfekt!

**Pull request merged**